data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,315 @@ mission "Coalition License Patch"
+	to offer
+		has "event: yottrite research break"
+	source
+		government "Coalition" "Heliarch"
+	destination "Pelubta Station"
+	on offer
+		conversation
+			`When you land, you receive a message from Lugglop, the Arach who you brought several Yottrite samples to.`
+			`	"Captain, some major findings, we have. Preparing a presentation about our research, to present in Ring of Wisdom, we are," he says. "Only, run out of pure samples, we have. Bring us one once more, would you?"`
+			choice
+				`	"Okay, I'll bring you a piece of Yottrite once again."`
+				`	"Sorry, but I'm not mining that any more."`
+					decline
+			`	He thanks you and says he and the rest of the team will be waiting in <destination>. "Moved here about halfway through the project, we did, to make use of the station's facilities."`
+				accept
+	on complete
+		outfit "Yottrite" -1
+		payment 220000
+		conversation
+			`Lugglop pays you <payment> as you deliver the Yottrite, accompanied by a Heliarch agent.`
+			`	"Finish preparing our presentation, we now must. If to help us further, you wish, in the spaceport meet us you must."`
+
+
+
+mission "Coalition Yottrite 9"
+	name "Yottrite Presentation"
+	description "Transport Lugglop and his team, alongside <cargo>, to <destination>, where they will share with the Coalition's scientific community their findings on Yottrite."
+	passengers 6
+	cargo "Yottrite Presentation Materials" 6.14
+	to offer
+		has "Coalition Yottrite 8: done"
+	source "Pelubta Station"
+	destination "Ring of Wisdom"
+	on offer
+		conversation
+			`You meet Lugglop in the spaceport and follow him to where his team is. You meet again with Nasilor, Alituri and Pichiie as the three of them oversee some workers place a few sealed metal crates inside a container and the Yottrite sample you just brought here into another.`
+			`	They detail how they have prepared all the material they needed and now just need a transport to <destination>.`
+			`	"If with us, you come, Captain, perhaps a better detailed description of the systems where the Yottrite was found, you could provide," Nasilor says.`
+			`	"<payment>, I would pay you," Lugglop says.`
+			choice
+				`	"Of course I'll take you there. Have the containers loaded into my ship."`
+					goto accept
+				`	"What discoveries have you made about it?"`
+					goto findings
+			label findings
+			`	"To share our findings with you, allowed we are not," Pichiie says.`
+			`	"Forgive us you must, Captain, but told to keep everything a secret, we were," Lugglop continues. "Perhaps after our presentation, public our findings could be made, though."`
+			label accept
+			`	They all follow you to your ship after the containers are ready to be loaded.`
+			`	After they are put into your cargo, Nasilor, Alituri and Pichiie head to their bunks, but as Lugglop is coming on board, you see that two Heliarch agents are accompanying him. He introduces you to the two of them, both Kimek, though you can't quite catch their names.`
```

Comment:
```suggestion
			`	After they are put into your cargo, Nasilor, Alituri, and Pichiie head to their bunks, but as Lugglop is coming on board, you see that two Heliarch agents are accompanying him. He introduces you to the two of them, both Kimek, though you can't quite catch their names.`
```
Why don't I catch their names? If they aren't important enough to be named for the player, just don't have them be introduced.